hummingbird/ml/operator_converters/skl_linear.py

```diff
@@ -0,0 +1,110 @@
+        if self.multi_class == "multinomial":
+            output = torch.softmax(output, dim=1)
+        elif self.regression:
+            if not self.binary_classification:
+                return output
+        else:
+            output = torch.sigmoid(output)
+            if not self.binary_classification:
+                output /= torch.sum(output, dim=1, keepdim=True)
+
+        if self.binary_classification:
+            output = torch.cat([1 - output, output], dim=1)
+
+        if self.perform_class_select:
+            return torch.index_select(self.classes, 0, torch.argmax(output, dim=1)), output
+        else:
+            return torch.argmax(output, dim=1), output
+
+
+def convert_sklearn_linear_model(operator, device, extra_config):
+    """
+    Converter for `sklearn.svm.LinearSVC`, `sklearn.linear_model.LogisticRegression`,
+        `sklearn.linear_model.SGDClassifier`, and `sklearn.linear_model.LogisticRegressionCV`
+
+    Args:
+        operator: An operator wrapping a `sklearn.svm.LinearSVC`, `sklearn.linear_model.LogisticRegression`,
+            `sklearn.linear_model.SGDClassifier`, or `sklearn.linear_model.LogisticRegressionCV` model
+        device: String defining the type of device the converted operator should be run on
+        extra_config: Extra configuration used to select the best conversion strategy
+
+    Returns:
+        A PyTorch model
+    """
+    classes = [0] if not hasattr(operator.raw_operator, "classes_") else operator.raw_operator.classes_
+
+    if not all([type(x) in [int, np.int32, np.int64] for x in classes]):
+        raise RuntimeError("hummingbird supports only integer labels for class labels.")
+
+    coefficients = operator.raw_operator.coef_.transpose().astype("float32")
+    intercepts = operator.raw_operator.intercept_.reshape(1, -1).astype("float32")
+
+    if hasattr(operator.raw_operator, "multi_class"):
+        if operator.raw_operator.multi_class == "ovr" or operator.raw_operator.solver in ["warn", "liblinear"]:
+            multi_class = "ovr"
+        else:
+            multi_class = "multinomial"
+    else:
+        multi_class = None
+
+    return SklearnLinearModel(coefficients, intercepts, classes, multi_class, device)
+
+
+def convert_sklearn_linear_regression_model(operator, device, extra_config):
+    """
+    Converter for `sklearn.linear_model.LinearRegression`
+
+    Args:
+        operator: An operator wrapping a `sklearn.linear_model.LinearRegression` model
+        device: String defining the type of device the converted operator should be run on
+        extra_config: Extra configuration used to select the best conversion strategy
+
+    Returns:
+        A PyTorch model
+    """
+
+    coefficients = operator.raw_operator.coef_.transpose().reshape(-1, 1).astype("float32")
+    intercepts = operator.raw_operator.intercept_.reshape(1, -1).astype("float32")
+
+    return SklearnLinearModel(coefficients, intercepts, [0], None, device, is_linear_regression=True)
```

Comment:
Can we set these as default params in the class and move device after intercepts? In this way for linear regression we only need to write `return SklearnLinearModel(coefficients, intercepts, device)`.